### PR TITLE
Update template pull_request_template.md

### PR DESCRIPTION
**Pull request by dgsudharsan**
Updating template description to include link to configuration schema for YANG module changes.

#### Why I did it
Updating template description to include link to configuration schema for YANG module changes. When reviewing yang models it becomes difficult to know the entire schema for the table for which yang is defined. Besides providing a clear picture, it will also help to document the config_db schema which is not upto date.

#### How I did it
Updating template file

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [x] 202012
- [x] 202106

### CODE REVIEW

**Review by liat-grozovik — DISMISSED** [review]

**Review by liat-grozovik — COMMENTED** [review]
could  you please add 202111 branch to the list?

**Review by liat-grozovik — DISMISSED** [review]

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -30,13 +30,19 @@
...
 - [ ] 202106
+- [ ] 202111
 
 #### Description for the changelog
 <!--
 Write a short (one line) summary that describes the changes in this
 pull request for inclusion in the changelog:
 -->
 
+#### Link to config_db schema for YANG module changes
```
Comment (praveen-li) [inline .github/pull_request_template.md:41]: Thanks for this PR. We can also add Page details.

Provide a link to config_db schema for the table for which YANG model
is defined.
Link should point to correct section on https://github.com/Azure/SONiC/wiki/Configuration.

**Review by praveen-li — COMMENTED** [review]
LGTM

File: .github/pull_request_template.md (hunk 1 above)
Comment (dgsudharsan) [reply .github/pull_request_template.md:41]: Done

**Review by praveen-li — APPROVED** [review]
LGTM

**Review by qiluo-msft — APPROVED** [review]

**Comment by qiluo-msft** [comment]
No pipeline [path filters](https://go.microsoft.com/fwlink/?linkid=2084226) matched the file changes.
Will override and merge.

**Pull request merged**